src/failover_website/index.html

```diff
@@ -1 +1,49 @@
-Hello, world.
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <!-- vendor.css contains styles which override bootstrap styling.  It must come after main.css -->
+    <link rel="stylesheet" href="static/css/main.css">
+    <link rel="stylesheet" href="static/css/vendor.css">
+    <link rel="stylesheet" type="text/css" href="http://cloud.typography.com/7429932/717646/css/fonts.css" />
+  </head>
+  <body>
+    <header class="main-header">
+      <nav class="nav-main clearfix">
+        <div class="nav-left" style="padding-left: 5px">
+          <div class="brand nav-left">
+            <img src="static/img/logo-treescount.png" alt="TreesCount! 2015 logo">
+          </div>
+        </div>
+      </nav>
+    </header>
+    <div class="overlay-menued"></div>
+    <div class="right-content">
+      <div class="page-home">
+        <div class="hero">
+          <div class="hero-background" style="background-image: url(static/img/background-aerial.jpg)">
+          </div>
+          <div class="hero-box">
+            <div class="container">
+              <div class="row">
+                <div class="col-sm-12">
+                  <div class="hero-content">
+                    <h1 class="hero-heading">Website Down</h1>
+                    <p>Our website is down — sorry everyone. NYC Parks invites you to join us in counting and mapping all of New York City's beautiful street trees. Come play a key role in planning the future of our urban forest.</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <footer class="visible-xs">
+        <div class="container">
+            <img class="section-about-logo pull-left" src="static/img/logo-nycparks.png" srcset="/static/img/logo-nycparks@2x.png 2x">
```

Comment:
Ah wget must not be able to handle this feature. I'll make sure the 2x graphics make it in too.